bindsnet/network/nodes.py

```diff
@@ -338,7 +339,9 @@ def __init__(self, n: Optional[int] = None, shape: Optional[Iterable[int]] = Non
         :param rest: Resting membrane voltage.
         :param reset: Post-spike reset voltage.
         :param refrac: Refractory (non-firing) period of the neuron.
+        :param lbound: Lower bound of the voltage.
         :param decay: Time constant of neuron voltage decay.
+        :param noise_std: Standard deviation of Gaussian noise to membrane potential. This helps exploration.
```

Comment:
I don't think this should be part of the package proper. If you want to have this in your custom subclasses, that's fine.

Reply:
I've removed noise functionality. Thanks.